GitExercise_1_Tests/UnitTest1.cs

```diff
@@ -0,0 +1,442 @@
+using System;
+using System.Collections.Generic;
+using GitExercise_1;
+using NUnit.Framework;
+
+
+namespace GitExercise_1_Tests
+{
+    [TestFixture]
+    public class PointTests
+    {
+
+        [Test]
+        public void Creating_Point_instance_with_empty_constructor()
+        {
+            // Arrange
+            Point P1;
+
+            // Act
+            P1 = new Point();
+
+            // Assert
+            Assert.IsInstanceOf<Point>(P1);
```

Comment:
It doesn't make sense to test code from previous line

Reply:
Ok. Thanks for your advices.